draft-irtf-cfrg-frost.md

```diff
@@ -857,46 +891,10 @@ parameters, to check that the signature share is valid using the following proce
     return l == r
 ~~~
 
-If any signature share fails to verify, i.e., if verify_signature_share returns False for
-any participant share, the Coordinator MUST abort the protocol for correctness reasons
-(this is true regardless of the size or makeup of the signing set selected by
-the Coordinator).
-Excluding one participant means that their nonce will not be included in the joint response `z`
```

Comment:
We should include this discussion that the resulting signature is invalid if any signature share is invalid (and the subset of valid signature shares will *not* create a valid signature) somewhere as it is request from a prior review: https://mailarchive.ietf.org/arch/msg/cfrg/ZkRsbR8nsadkSZ5KAJz4wS-h0IM/

Reply:
This is already stated in the PR. Please suggest clarifications if you think further clarifications are needed.